public/app/features/dashboard/api/v1.ts

```diff
@@ -126,6 +130,14 @@ export class K8sDashboardAPI implements DashboardAPI<DashboardDTO, Dashboard> {
         },
       };
 
+      if (dash.metadata.annotations?.[AnnoKeyManagerKind]) {
+        const allowEdits =
+          dash.metadata.annotations?.[AnnoKeyManagerAllowsEdits] === 'true' ||
+          dash.metadata.annotations?.[AnnoKeyManagerKind] === ManagerKind.Repo;
+        result.meta.provisioned = allowEdits;
+        result.meta.provisionedExternalId = dash.metadata.annotations?.[AnnoKeySourcePath];
```

Comment:
this is used by:
https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/saving/SaveProvisionedDashboardForm.tsx#L52